adalib/ada/datasets/office_dataset_access.py

```diff
@@ -20,10 +20,10 @@ def get_accesses(source: "Office31Dataset", target: "Office31Dataset", data_path
 
 class Office31DatasetAccess(DatasetAccess):
     def __init__(self, domain, data_path):
+        super().__init__(n_classes=31)

```

Comment:
What happens if I call the constructor with n_classes < 31? or n_classes > 31? As far as I remember, it may simply build a classifier with that number of classes (but it won't affect the data loader).
We might want to let the user change the number of classes at some point, but I'm not sure we are handling it meaningfully for now.

Reply:
My bad, looks good.